tpl/form/login.tpl

```diff
@@ -1,6 +1,6 @@
 [{oxscript include="js/libs/jqBootstrapValidation.min.js" priority=10}]
 [{oxscript add="$('input,select,textarea').not('[type=submit]').jqBootstrapValidation();"}]
-<div class="card">
+<div class="card" style="height:auto">
```

Comment:
Please use a css class or an id, but no inline css.

Reply:
Are you sure? This is only necessary in this one, very specific instance. I didn't want to clutter the main CSS file with it.

Or would it be OK to put it into an oxstyle block at the beginning of the .tpl file?

Reply:
my opinion: It is bad practice.  
try to in the theme css files, if there is a already defined class, with the required style.

Reply:
Alright, it is done ;-)

I also removed user_checkout_noregistration.tpl, since this checkout workflow doesn't need it any more.